app/main.py

```diff
@@ -1 +1,26 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> str | None:
+    if not command.startswith("mv ") or len(command.split()) != 3:
+        return "Wrong command"
+
+    _, source_file, new_file, *_ = command.split()
```

Comment:
Firstly, perform this operation and save the result into the variable 'result'. Then, you can check if the length of 'result' is 3 and if the first element is 'mv'. If so, unpack the result into variables.